orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java

```diff
@@ -240,7 +240,11 @@ public Work updateWork(String orcid, Work work, boolean applyAPIValidations) {
         
         if (applyAPIValidations) {
             activityValidator.validateWork(work, sourceEntity, false, applyAPIValidations, workEntity.getVisibility());
-            List<Work> existingWorks = this.findWorks(orcid, System.currentTimeMillis());            
+            
+            Date lastModified = profileEntityManager.getLastModified(orcid);
+            long lastModifiedTime = (lastModified == null) ? 0 : lastModified.getTime();
+            List<Work> existingWorks = this.findWorks(orcid, lastModifiedTime);       
```

Comment:
Ok, so, lets go with this change and refactor later for best performance.

@TomDemeranville please fix the merge conflict